### PR TITLE
Add Huawei CSI support

### PR DESCRIPTION
**Pull request by wuxueyi19**
Add Huawei CSI support for enterprise storage and could storage.

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [vSphere](https://github.com/kubernetes-sigs/vsphere-csi-driver) | `vsphere.csi.vmware.com` | v1.0 | A Container Storage Interface (CSI) Driver for VMware vSphere | Persistent | Read/Write Single Pod | Yes | Raw Block
 [YanRongYun](http://www.yanrongyun.com/) | ? | v1.0 | A Container Storage Interface (CSI) Driver for YanRong YRCloudFile Storage  | Persistent | Read/Write Multiple Pods | Yes | 
 [Zadara-CSI](https://github.com/zadarastorage/zadara-csi) | `csi.zadara.com` | v1.0, v1.1 | A Container Storage Interface (CSI) plugin for Zadara VPSA Storage Array & VPSA All-Flash  | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot, Expansion, Cloning
+[Huawei Storage CSI](https://github.com/Huawei/eSDK_K8S_Plugin) | `csi.huawei.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Huawei Storage](https://support.huawei.com/enterprise/en/category/enterprise-storage-pid-1482606575525?submodel=21430818, https://support.huawei.com/enterprise/en/category/cloud-storage-pid-1482615290338?submodel=21430817) | Persistent | Read/Write Multiple Pod | Yes |
```
Comment (msau42) [inline book/src/drivers.md]: Can you put this in alphabetical order in the list?

Comment (msau42) [reply book/src/drivers.md]: Also on the github page, can you provide instructions on how to install the csi driver?

Comment (wuxueyi19) [reply book/src/drivers.md]: 1. The new added content is in alphabetical now.
2. The document "User Guide.pdf" has been added to https://github.com/Huawei/eSDK_K8S_Plugin/tree/master/doc.

File: book/src/drivers.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Hammerspace CSI](https://github.com/hammer-space/csi-plugin) | `com.hammerspace.csi` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for Hammerspace Storage | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot
 [Hitachi Vantara](https://knowledge.hitachivantara.com/Documents/Adapters_and_Drivers/Storage_Adapters_and_Drivers/Containers) | `com.hitachi.hspc.csi` | v1.0 | A Container Storage Interface (CSI) Driver for VSP series Storage | Persistent | ? | ? | 
 [HPE](https://github.com/hpe-storage/csi-driver) | `csi.hpe.com` | v0.3, v1.0, v1.1 | A Container Storage Interface (CSI) driver from HPE | Persistent or Ephemeral | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Cloning
+[Huawei Storage CSI](https://github.com/Huawei/eSDK_K8S_Plugin) | `csi.huawei.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Huawei Storage](https://support.huawei.com/enterprise/en/category/enterprise-storage-pid-1482606575525?submodel=21430818, https://support.huawei.com/enterprise/en/category/cloud-storage-pid-1482615290338?submodel=21430817) | Persistent | Read/Write Multiple Pod | Yes |
```
Comment (saad-ali) [inline book/src/drivers.md]: Why are there two links for `Huawei Storage`? That breaks the markdown.

Comment (wuxueyi19) [reply book/src/drivers.md]: Now there has only one link. Please check. Thank you!

**Comment by saad-ali** [comment]
Please squash commits. Otherwise LGTM

**Comment by wuxueyi19** [comment]
> Please squash commits. Otherwise LGTM

Sorry， I don't understand your meaning. What should I do next?

**Comment by msau42** [comment]
To squash commits, I do something similar to this tutorial:
https://www.mikulskibartosz.name/git-fixup-explained/

I do an interactive rebase, and then change the later commits to be "fixup". This will merge the later commits into the inital commit.

**Comment by wuxueyi19** [comment]
@saad-ali @lpabon I see that changes and review are requested. Could you advise how to complete this pull request please? Thank you!

**Comment by msau42** [comment]
@wuxueyi19 the ask is to squash your comments into 1. Can you try out the [instructions](https://github.com/kubernetes-csi/docs/pull/263#issuecomment-589909887)?

File: book/src/drivers.md [hunk 3]
```diff
@@ -36,7 +36,8 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
...
 [Hammerspace CSI](https://github.com/hammer-space/csi-plugin) | `com.hammerspace.csi` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for Hammerspace Storage | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot
 [Hitachi Vantara](https://knowledge.hitachivantara.com/Documents/Adapters_and_Drivers/Storage_Adapters_and_Drivers/Containers) | `com.hitachi.hspc.csi` | v1.0 | A Container Storage Interface (CSI) Driver for VSP series Storage | Persistent | ? | ? | 
-[HPE](https://github.com/hpe-storage/csi-driver) | `csi.hpe.com` | v0.3, v1.0, v1.1 | A Container Storage Interface (CSI) driver from HPE | Persistent and Ephemeral | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Cloning
+[HPE](https://github.com/hpe-storage/csi-driver) | `csi.hpe.com` | v0.3, v1.0, v1.1 | A Container Storage Interface (CSI) driver from HPE | Persistent or Ephemeral | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Cloning
```
Comment (xing-yang) [inline book/src/drivers.md:39]: Is this change from `Persistent and Ephemeral` to `Persistent or Ephemeral` for HPE driver needed for this PR?

**Comment by wuxueyi19** [comment]
If there are no other problems, Could you merge it?

**Comment by xing-yang** [comment]
@wuxueyi19 have you tried @msau42's instructions to squash the commits into 1?  Currently there are 8 commits: https://github.com/kubernetes-csi/docs/pull/263/commits.  The ask is to squash them so that there will be just 1 commit.

**Comment by wuxueyi19** [comment]
@xing-yang Thank you for your kind attention to this. We tried the instructions but did not succeed. Now we are working on it again.

**Comment by wuxueyi19** [comment]
See more at https://github.com/kubernetes-csi/docs/pull/303.

**Pull request closed without merging**